lib/Auth/Process/SwitchAuth.php

```diff
@@ -117,20 +119,37 @@ public function process(&$state)
                 && !AuthnContextHelper::MFAin([$upstreamContext])
         ); // switch to MFA if preferred and not already done if we handle the proxy mode
 
+        $assurance = '';
+        if (in_array(AuthSwitcher::SFA, $usersCapabilities, true) && in_array(
+            AuthSwitcher::MFA,
+            $usersCapabilities,
+            true
+        )) {
+            $assurance = AuthSwitcher::MFA;
```

Comment:
```suggestion
            $maxUserCapability = AuthSwitcher::MFA;
```
I think a more descriptive name for the variable would be something like `$maxUserCapability`